src/Raven.Server/Documents/ETL/EtlLoader.cs

```diff
@@ -229,6 +240,15 @@ private void HandleChangesSubscriptions()
                         else
                             connectionStringNotFound = true;
                         break;
+                    case EtlType.Elasticsearch:
```

Comment:
The first version was just `Elastic`, then we switched to `Elasticsearch` because they are using that phrase on their website.
But I will change that.

Reply:
Using `ElasticSearch` was my initial reaction as well...
But since the formal name they use all over is `Elasticsearch` then we probably better follow that...